Product/Production/Gateway/AdminGUIWebservices/src/main/java/gov/hhs/fha/nhinc/admingui/loadtestdata/LoadTestDataManagement.java

```diff
@@ -0,0 +1,507 @@
+        if (!isId(request.getPatientId())) {
+            return buildSimpleResponse(Boolean.FALSE, ACT_FAIL);
+        }
+        LoadTestDataSimpleResponseMessageType response = new LoadTestDataSimpleResponseMessageType();
+        for (Personname item : loadTestDataService.getAllPersonnamesBy(request.getPatientId())) {
+            response.getPersonnameList().add(item.getPersonnameType());
+        }
+        return response;
+    }
+
+    @Override
+    public LoadTestDataSimpleResponseMessageType listAllPhonenumber(ListAllPhonenumberRequestMessageType request) {
+        if (!isId(request.getPatientId())) {
+            return buildSimpleResponse(Boolean.FALSE, ACT_FAIL);
+        }
+        LoadTestDataSimpleResponseMessageType response = new LoadTestDataSimpleResponseMessageType();
+        for (Phonenumber item : loadTestDataService.getAllPhonenumbersBy(request.getPatientId())) {
+            response.getPhonenumberList().add(item.getPhonenumberType());
+        }
+        return response;
+    }
+
+    @Override
+    public LoadTestDataSimpleResponseMessageType saveAddress(SaveAddressRequestMessageType request) {
+        Patient patient = loadTestDataService.getPatientBy(request.getAddress().getPatientId());
+        if(null != patient){
+            try{
+                Address record = new Address(request.getAddress(), patient);
+                loadTestDataService.saveAddress(record);
+                if (isId(record.getAddressId())) {
+                    return buildSimpleResponse(Boolean.TRUE, ACT_SUCCESSFUL, record.getAddressId());
+                }
+            } catch (LoadTestDataException ex) {
+                LOG.error("error while saving patient address: {}", ex.getLocalizedMessage(), ex);
+            }
+        }
+        return buildSimpleResponse(Boolean.FALSE, ACT_FAIL);
+    }
+
+    @Override
+    public LoadTestDataSimpleResponseMessageType saveDocument(SaveDocumentRequestMessageType request) {
+        if (null != request) {
+            try {
+                DocumentMetadata record = new DocumentMetadata(request.getDocument());
+                loadTestDataService.saveDocument(record);
+                if (isId(record.getDocumentid())) {
+                    return buildSimpleResponse(Boolean.TRUE, ACT_SUCCESSFUL, record.getDocumentid());
+                }
+            } catch (LoadTestDataException ex) {
+                LOG.error("error while saving document: {}", ex.getLocalizedMessage(), ex);
+            }
+        }
+        return buildSimpleResponse(Boolean.FALSE, ACT_FAIL);
+    }
+
+    @Override
+    public LoadTestDataSimpleResponseMessageType saveEventCode(SaveEventCodeRequestMessageType request) {
+        DocumentMetadata meta = loadTestDataService.getDocumentBy(request.getEventCode().getDocumentid());
+        if (null != meta) {
+            try {
+                EventCode record = new EventCode(request.getEventCode(), meta);
+                loadTestDataService.saveEventCode(record);
```

Comment:
Should use the result to determine if it passed or failed - Same issue applies for most of these